packages/components/ResultData.tsx

```diff
@@ -0,0 +1,42 @@
+import { Flex, Grid } from '@chakra-ui/react';
+
+import ResultCard from './resultCard';
```

Comment:
This is breaking the CI. Since the PR is looking to main as base, this import can't exist.